README.md

```diff
@@ -16,23 +16,31 @@
   <img src="https://img.shields.io/twitter/follow/maptiler?style=social"></img>
 </p>
 
-# MapTiler Geocoding control for MapTiler SDK, MapLibre GL JS, Leaflet and OpenLayers
+# MapTiler Search and Geocoding control for MapTiler SDK, MapLibre GL JS, Leaflet and OpenLayers
 
-## About
+## What?
 
-A _Geocoding control_ for [MapTiler SDK](https://github.com/maptiler/maptiler-sdk-js),
-[MapLibre GL JS](https://github.com/maplibre/maplibre-gl-js),
-[Leaflet](https://leafletjs.com) and [OpenLayers](https://openlayers.org) utilizes [MapTiler Cloud Geocoding
-API](https://www.maptiler.com/cloud/geocoding/). With this control, users of
-mapping application can find any place on Earth (States, Cities, Streets, Addresses, POIs, ...) down
-to the address level, restrict the search area to a specific country, highlight
-searched results on the map, autocomplete words while typing, and much more.
+The *MapTiler Geocoding control* implements a powerful search box in your maps or online forms, enabling your application users to find any place on Earth down to individual addresses. Use the search box control with [MapTiler SDK JS](https://docs.maptiler.com/sdk-js/) (or other map libraries like [Leaflet](https://docs.maptiler.com/leaflet/), [MapLibre GL JS](https://github.com/maplibre/maplibre-gl-js), [OpenLayers](https://docs.maptiler.com/openlayers/)).
+
+## Why?
+
+The _Geocoding control_ uses the [MapTiler Geocoding API](https://www.maptiler.com/cloud/geocoding/). You can use the API directly from your web or backend applications or use the [API Client JS](https://docs.maptiler.com/client-js/) library.
+
+With this control, users of
+mapping application can:
+
+* Find any place on Earth (States, Cities, Streets, Addresses, POIs, ...) down
+to the address level
+* Restrict the search area to a specific country, bounding box, or proximity
+* Highlight searched results on the map (marker or full geometry) 
+* Autocomplete words while typing
```

Comment:
Plesae add bullet point with reverse geocoding. Something like

* Find and identify objects using coordinate pair or single mouse click